### DISCUSSION TITLE
Fusion Dev 초대 메일을 받지 못했습니다.

### DISCUSSION THREAD

**Discussion by srh1004heejin** [fusiondevkr/pl900#5] [1 Q&A] [answered]
안녕하세요. 이희진입니다.
M365 개발자 계정 생성, Power Apps 개발자 라이선스 등록 후 새로 생성된 개발자 M365 홈포털에 접속했는데 라이선스가 할당되지 않아 라이선스 할당 후 다시 Outlook 앱에 접속하니  Fusion DeV 초대 수락 메일이 들어오지 않습니다.
![image](https://user-images.githubusercontent.com/40831091/125953696-18fca072-afc9-49e0-b661-99ff61505dd6.png)
**제가 생성한 Microsoft365 개발자 계정은 heejin@plsrh1004.onmicrosoft.com 입니다.**
수동으로 초대 부탁 드리겠습니다.
고맙습니다.

**Comment by sesme100** [comment]
안녕하세요 "Fusion DEV Korea가 조직내에서 애플리케이션에 엑세스 하도록 초대함"의 이메일은 Azure Acitve Directory에서 자동 생성 되어 전달 되는 메일입니다. 저희가 자동화로 만든 흐름을 보니 "AAD에 게스트 사용자 추가" 액션에서 잘 실행 되었습니다. 
그에 따라 AAD와 아웃룩의 문제로 보입니다. 초대 수락 링크를 보내는 이메일을 수동으로 보낼 수 있는지 한번 확인 해 보겠습니다.

**Comment by justinyoo** [comment]
`plsrh1004.onmicrosoft.com` 요 테넌트가 M365 개발자 계정 테넌트인데요, 여기로 로그인해서 메일박스에 보면 초대 링크가 있을 거예요. 혹시 거기 안 보이면 다시 한번 알려주세요.

**Reply by justinyoo** [reply]
@srh1004heejin 방금 다시 보냈는데요,  heejin@plsrh1004.onmicrosoft.com 요 이메일로 접속해서 메일박스 한 번 확인해 보세욥!

**Reply by choiyoon07** [reply]
@justinyoo 저도 초대 이메일 다시 부탁드립니다.

**Comment by srh1004heejin** [comment, marked as answer]
안녕하세요.
어제 라이선스가 할당되지 않아서 혹시 반영되지 않았을거 같아 질문 남겼습니다.
오늘 오전에 확인해보니 메일이 와서 [초대 수락]했습니다.
고맙습니다.

**Comment by danielkim360** [comment]
@justinyoo Fusion DEV Korea이(가) 조직 내에서 애플리케이션에 액세스하도록 초대함 메일을 'daniel@ki0m36.onmicrosoft.com' 여기로 받지 못했습니다. 발송 부탁드리겠습니다!

**Reply by justinyoo** [reply]
방금 보내드렸어요. 다시 한 번 확인해 보세요.

**Reply by danielkim360** [reply]
감사합니다! 잘 받았습니다.

**Comment by srh1004heejin** [comment]
메일이 잘 못 발송된 듯 합니다.
확인 부탁드리겠습니다.
